overturemaps/cli.py

```diff
@@ -27,17 +27,34 @@ def get_writer(output_format, path, schema):
     elif output_format == "geojsonseq":
         writer = GeoJSONSeqWriter(path)
     elif output_format == "geoparquet":
-        # Update the geoparquet metadata to remove the file-level bbox which
-        # will no longer apply to this file. Since we cannot write the field at
-        # the end, just remove it as it's optional. Let the per-row bounding
-        # boxes do all the work.
+        # remove geom extents bbox and add covering spec for spatial filters
         metadata = schema.metadata
+        # extract row metadata
+        row_meta = json.loads(metadata[b"org.apache.spark.sql.parquet.row.metadata"])
```

Comment:
This should use the schema directly here rather than the spark metadata. I think you can just do something like: `has_bbox = "bbox" in schema.names` or something like that.